geotrellis/tree-prioritization-demo
# Remove summary service

* Move code from `common` project to `tile` project
* Remove `summary`, `common`, and `combined` projects
* Update `build.scala` to build just the `tile` project
* Remove references to Spark jobserver
* Simplify `rebuild.sh` and `endpoints.sh`
* Update version to 1.2.0

Testing:

* Check out OpenTreeMap/otm-cloud#319
* Destroy and recreate your modeling VM
* Verify that tiles still load
* Create a scenario with a tree and a distribution
* Verify that popups still work (i.e. don't hit the summary service)
* Verify that the scenario still runs

Depends on OpenTreeMap/otm-cloud#319
Connects #124
